core/state_processor.go

```diff
@@ -143,24 +141,8 @@ func applyTransaction(msg *Message, config *params.ChainConfig, gp *GasPool, sta
 		statedb.AddBalance(result.BurntContractAddress, result.FeeBurnt)
 	}
 
+	// TODO(raneet10) Double check
 	statedb.AddBalance(evm.Context.Coinbase, result.FeeTipped)
-	output1 := new(big.Int).SetBytes(result.SenderInitBalance.Bytes())
-	output2 := new(big.Int).SetBytes(coinbaseBalance.Bytes())
-
-	// Deprecating transfer log and will be removed in future fork. PLEASE DO NOT USE this transfer log going forward. Parameters won't get updated as expected going forward with EIP1559
-	// add transfer log
-	AddFeeTransferLog(
-		statedb,
-
-		msg.From,
-		evm.Context.Coinbase,
-
-		result.FeeTipped,
-		result.SenderInitBalance,
-		coinbaseBalance,
-		output1.Sub(output1, result.FeeTipped),
-		output2.Add(output2, result.FeeTipped),
-	)
```

Comment:
Need to double check the correctness of this. Tipping is already skipped in `ApplyMessageNoFeeBurnOrTip(evm, *msg, gp, interruptCtx)` (line 132) above. If we don't tip miner at this point, the miner won't get tipped at all. cc @pratikspatil024

Reply:
Yeah I don't remember exactly why the logging was removed but I can restore it. Will it have an effect on miners not getting tip fee ?

Reply:
Fee is tipped to miner in line 145. If lines from 147 to 163 are removed, the tipping log will be missing, which results in mismatched receipt hash.

Reply:
Ahh yes makes sense!